addons/exthttpaccess/addon.manifest.yaml

```diff
@@ -26,6 +26,10 @@ spec:
             shorthand: p
             default: ""
             description: HTTP Proxy
+          - name: force-use-alternative-ports-if-needed
```

Comment:
It would be nice to have a simple flag name e.g. **--force-use-ports** and explain further in the usage notes/description. Too long flag is error prone for user. What do you think @gabriel-rosso ?

Reply:
That flag name conveys the real intention of what the user does/gets when using it.
Using a shorter name would expect that the user remembers what he read in the help description (when using it the first time, and when he comes back to the code some time later).

The drawbacks of shorter names:
'--force-use-ports': the addon is not deliberately changing the ports.
'--force': the addon is not forcing the enabling on the usual HTTP(S) ports.

For a shorter name there is the shortcut '-f'

Examples of long flag names are:
--allow-missing-template-keys (kubectl create)
--discovery-token-unsafe-skip-ca-verification (kubeadm join)

Reply:
@gabriel-rosso Yes, kubectl is notorious for long flag names which we should avoid. 
Can this help? In this case, it clearly indicates addon will switch to alternative ports when 80/443 are busy.
`-a --auto-alt-ports   Automatically switch to alternative ports 8080/8443 if 80/443 are busy without user confirmation`

Reply:
@mahenarayan What about `--autoconfirm-alt-ports` ?